bears/css/StyleLintBear.py

```diff
@@ -0,0 +1,32 @@
+import os
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylelint',
+        output_format='regex',
+        output_regex=r'.+\n\s(?P<line>\d+):(?P<column>\d+)'
+                     r'\s+✖\s(?P<message>.+)')
+class StyleLintBear:
+    """
+    Checks the code with stylelint. This will run stylelint over each file
+    separately.
+
+    Detect errors and potential problems in CSS code and to enforce
+    appropriate coding conventions. For example, problems like syntax errors,
+    invalid color codes etc can be detected.
+
+    For more information on the analysis visit <http://stylelint.io/>
+    """
+    LANGUAGES = {'CSS'}
+    REQUIREMENTS = {NpmRequirement('stylelint')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Syntax', 'Unused Code', 'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         stylelint_config_file: str=os.devnull):
+
+        return '--config=' + stylelint_config_file, filename
```

Comment:
it won't run unless the user provide a config file ( or that there is a .stylelintrc in the cur dir ) 
so maybe you should generate that for the user